### PR TITLE
Add online validation for connected components

### PR DESCRIPTION
**Pull request by fonurr**
- For projects and styleguides
- For Zeplin components

### CODE REVIEW

File: src/common/vscode/diagnostic/diagnosticsUtil.ts [hunk 1]
```diff
@@ -2,18 +2,46 @@ import * as vscode from "vscode";
...
+ * @param document A text document to create diagnostics on.
+ * @param severity Severity of diagnostics to be created
+ */
+function createDiagnostics(
+    propertyName: string,
+    message: string,
+    document: vscode.TextDocument,
+    severity?: vscode.DiagnosticSeverity
+): vscode.Diagnostic[];
+
```
Comment (konukselim) [inline src/common/vscode/diagnostic/diagnosticsUtil.ts:18]: already in here 👇

Comment (konukselim) [reply src/common/vscode/diagnostic/diagnosticsUtil.ts:18]: I just realized `string` and `string[]` difference.

Comment (fonurr) [reply src/common/vscode/diagnostic/diagnosticsUtil.ts:18]: there is a small change in the first parameter there, but I made it wrong appearently, I'll fix it

Comment (fonurr) [reply src/common/vscode/diagnostic/diagnosticsUtil.ts]: I fixed the documentation

**Review by konukselim — APPROVED** [review]

**Comment by fonurr** [comment]
Force pushed for merging fixups by running: git rebase -i --autosquash origin/master

**Pull request merged**